### ISSUE TITLE
I should set up for every project

### ISSUE THREAD

**Issue by soyking** [iamthemovie/GoSublimeAutocompleteUpdate#1]
Am I right?

**Comment by iamthemovie** [comment]
Yes but if I'm honest you shouldn't need to use this anymore as the packages for Go in sublime and atom are pretty good now!

**Comment by soyking** [comment]
But ... I still like Sublime ~ lol
I can't get autocomplete when a struct with anonymous fields
So do you know how to fix it ?

**Comment by iamthemovie** [comment]
Unsure, the reason I wrote this was because go build / go code was not running until I saved the file I was working in which is what caused a lack of autocomplete for code I was writing in real time. 

I have since switched to Atom / Vim for my golang projects as the packages are better supported.

Does GoSublime not work?

**Comment by soyking** [comment]
I see! Because my project's dependences are in `godep` folder. Then after saving GoSublime can't `go install` the package. Have your ever use `godep`? Maybe it can work on Atom / Vim.
Thank you anyway.

**Comment by iamthemovie** [comment]
No I haven't, not yet anyway. I tend to use gopkg.in for my dependency management.

I've never had an issue with the go-plus plugin for atom, it's constantly being updated and very well supported :)
